ci_env.yml

```
@@ -13,5 +13,5 @@ dependencies:
   - wearables 
   - idyntree 
   - matio-cpp 
```

```suggestion
  - libmatio-cpp 
```

Problem similar to robometry, eventually it will be fixed by https://github.com/conda-forge/libmatio-cpp-feedstock/pull/7 .